sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala

```diff
@@ -127,11 +128,10 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
   def sendCoinsToAddress: Route = (post & path("sendCoinsToAddress")) {
     entity(as[ReqSendCoinsToAddress]) { body =>
       // lock the view and try to create EvmTransaction
-      // TODO also account for gas fees
       applyOnNodeView { sidechainNodeView =>
         val valueInWei = ZenWeiConverter.convertZenniesToWei(body.value)
         val destAddress = body.to
-        val gasPrice = sidechainNodeView.getNodeState.getBaseFee // TODO actual gas implementation
+        val gasPrice = sidechainNodeView.getNodeState.getBaseFeePerGas.add(BigInteger.valueOf(200))
```

Comment:
No special reason, it is a default value I choose just to be above the base fee. This code will be changed as soon as a it will be meged with Sara's branch which adds specific parameters to this command:
```
        if (body.gasInfo.isDefined) {
          gasPrice = body.gasInfo.get.maxFeePerGas
          gasLimit = body.gasInfo.get.gasLimit
        }
```